contracts/core/Dispatcher.sol

```diff
@@ -161,11 +197,7 @@ contract Dispatcher is OwnableUpgradeable, UUPSUpgradeable, IDispatcher {
         string[] calldata connectionHops,
         CounterParty calldata counterparty,
         Ics23Proof calldata proof
-    ) external {
-        if (bytes(counterparty.portId).length == 0) {
-            revert IBCErrors.invalidCounterPartyPortId();
-        }
-
```

Comment:
We are doing extra checks on open-ack, any reason why we don't have any here?

Reply:
I think it's mainly because openTry and openInit don't modify the contract state; so I was thinking we don't do checks there to save on gas